protos/class.proto

```diff
@@ -11,19 +11,21 @@ service PresenterClassService {
     rpc DeleteClass (ClassDeleteRequest) returns (Status);
 
     rpc StartClass (ClassStartRequest) returns (stream ChatMessage);
-    rpc DelegateQuizQuestion (ClassEvent) returns (Status);
+    rpc DelegateQuizQuestion (QuizQuestion) returns (Status);
+    rpc GetQuizQuestionStatistics (QuizQuestion) returns (stream QuizQuestionStatistics);
     rpc EndClass (ClassEndRequest) returns (Status);
 }
 
 service StudentClassService {
-    rpc JoinClass (JoinClassRequest) returns (stream ClassEvent);
+    rpc JoinClass (JoinClassRequest) returns (stream QuizQuestion);
```

Comment:
Widac ze jest tu problem, co ma dostac uczen po dolaczeniu do zajec, moze tylko status, bo uczen moze podac bledny kod na przyklad, to co wtedy? dostanie zamkniety stream i trzeba bedzie robic jakies czary zeby to obslugiwac

wiec moze po poprawnym dolaczeniu osobne metody cos w stylu `RequestQuizes` i `RequestChatMessages` chociaz co do tego drugiego to nie jestem pewien czy tu nie trzeba jakos inaczej zrobic - moze w czacie

Reply:
rozumiem, że podczas
```    rpc JoinClass (JoinClassRequest) returns (Status);```
chcesz autentykować

a podczas 
```rpc JoinQuiz (JoinQuizRequest) returns (stream QuizQuestion);```
tylko odbierać stream

czyli

```
message JoinQuizRequest {
    User user = 1;
}
```

Reply:
wtedy chyba nie będzie potrzeby robienia _RequestChatMessages_ skoro już jesteśmy zalogowani?

Reply:
Możliwe że nie będzie potrzebne